frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -314,7 +367,7 @@ export const OperatorHubTileView: React.FC<OperatorHubTileViewProps> = (props) =
   return (
     <>
       <TileViewPage
-        items={props.items}
+        items={itemsByArchAndOS}
```

Comment:
Note that if I deliberately force a mismatch of `darwin`, then `<TileViewPage>` can get an empty array, which throws:

```
in TileViewPage (created by OperatorHubTileView)
    in OperatorHubTileView (created by OperatorHubList)
    in div (created by Data)
    ....
Stack Trace: 
./public/components/utils/tile-view-page.jsx/TileViewPage.prototype.render@http://localhost:9000/static/catalog~operator-hub-bc1ea21c06fb6f783133.js:622:1
finishClassComponent@http://localhost:9000/static/vendors~main-ee7ae4b95f81d76d0248.js:225357:31
updateClassComponent@http://localhost:9000/static/vendors~main-ee7ae4b95f81d76d0248.js:225312:24
beginWork$1@http://localhost:9000/static/vendors~main-ee7ae4b95f81d76d0248.js:226823:16
....
``` 
I need to fix this issue yet.